docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[ml-dataframes]]
+=== Data frames
+++++
+<titleabbrev>Dataframes</titleabbrev>
```

Comment:
This titleabbrev isn't needed unless you want a different title in the table of contents than on the page itself.

Reply:
I deleted it.